tests/BehatProvidingTraitTest.php

```diff
@@ -32,16 +32,19 @@ class BehatProvidingTraitTest extends TestCase {
   /**
    * @Then getProvidedScenario returns a scenario with the title :title
    */
-  public function getProvidedScenarioGets($title) {
-    $this->assertInstanceOf(ScenarioInterface::class, $this->getProvidedScenario());
+  public function getProvidedScenarioGets($title): void
+  {
+      $this->markTestSkipped('Not implemented yet.');
```

Comment:
Why mark the test as skipped? I'm ont sure what it means, but I'm concened it might weaken our test coverage.